bareMetalC/tiled_matmul_ws_perf.c

```diff
@@ -78,6 +69,16 @@ int main() {
     static full_t gold_full[MAT_DIM_I][MAT_DIM_J];
     static elem_t gold[MAT_DIM_I][MAT_DIM_J];
 
+
+    tiled_matmul_auto(MAT_DIM_I, MAT_DIM_J, MAT_DIM_K,
```

Comment:
You added an extra `tiled_matmul_auto` here.

Could you just reverse the changes to this file? I've added your changes to the `gemmini/conv-fsm2` branch, which is already a WIP PR that should be merged in a few days.

Reply:
This is to warm up the cache, which is necessary in the way the ci runs these programs

Reply:
But I can revert these changes for now. 
When you implement these changes in that branch, can you add this additional cache warmup? Otherwise the results that ci reports will be really bad

Reply:
Sure, I can add that

Reply:
Wouldn't the cache already be warmed though by the `static` array being zeroed-out in memory?

Reply:
Depends on the binary load procedure. In VCS simulation with `run-binary`, then it would be warmed up. In all other methods, it would not be warmed up.